pages/admin/index.vue

```diff
@@ -0,0 +1,48 @@
+<template>
+  <div class="d-flex align-center justify-center" style="height: 100vh">
```

Comment:
why vh?

Reply:
because it should take the full height of the screen

Reply:
You used `dvh` in another place, why the difference?

Reply:
ah, because I was not sure, if we should implement such new CSS properties, that some IDEs and some browsers don't support.

But I updated it to `dvh` since the whole template uses the latest features.